VimeoNetworking/Sources/Models/VIMQuantityQuota.m

```diff
@@ -39,8 +39,8 @@ @implementation VIMQuantityQuota
 
 - (void)didFinishMapping
 {
-    self.canUploadHd = [self.hd boolValue];
-    self.canUploadSd = [self.sd boolValue];
+    self.canUploadHd = [self.hd respondsToSelector:@selector(boolValue)] ? [self.hd boolValue] : NO;
+    self.canUploadSd = [self.sd respondsToSelector:@selector(boolValue)] ? [self.sd boolValue] : NO;
```

Comment:
I think we ran into a situation where we weren't getting expected values from the API for`NSNumber *hd` and ` NSNumber *sd`. I don't know what it was actually sending but whatever it was it couldn't handle being asked about its `boolValue` which was causing the 💥